sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -51,6 +51,7 @@
 public abstract class IntegrationTestBase extends TestBase {
     protected static final Duration OPERATION_TIMEOUT = Duration.ofSeconds(30);
     protected static final Duration TIMEOUT = Duration.ofSeconds(60);
+    protected static final Duration LOCK_RENEW_TIMEOUT = Duration.ofSeconds(15);
```

Comment:
This is only used in one place, move it to the method instead of making it a member variable.